### DISCUSSION TITLE
图片生成请求失效

### DISCUSSION THREAD

**Discussion by Fuckingcrazy-X** [juzeon/SydneyQt#254] [General]
![image](https://github.com/juzeon/SydneyQt/assets/88836287/80d8938d-13b9-4fd5-8486-4c29b93bf53e)
让它生成图片，似乎出现了某种错误，这是为何？

**Comment by juzeon** [comment]
通常是提示词或生成的图像触发了敏感检测。尝试换个提示词或用Bing官网的。
